androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/TopControls.kt

```diff
@@ -24,20 +29,27 @@ fun TopControls(
             .fillMaxWidth()
             .padding(horizontal = Dimensions.large)
     ) {
-        if (title != null && shouldShowContent) {
-            Text(
-                text = title,
-                maxLines = 1,
-                overflow = TextOverflow.Ellipsis,
-                style = MaterialTheme.typography.titleLarge.copy(
-                    color = Color.White
-                ),
-                modifier = Modifier.align(Alignment.CenterStart)
-            )
+        if (shouldShowContent) {
+            if (title != null) {
+                Text(
+                    text = title,
+                    maxLines = 1,
+                    overflow = TextOverflow.Ellipsis,
+                    style = MaterialTheme.typography.titleLarge.copy(
+                        color = Color.White
```

Comment:
I think you should use `MaterialTheme.colorScheme`

Reply:
it was like this before so I'll leave it like that. I think theres no problem because the video control UI is always white regardless.